applications/conference/src/conference_maintenance.erl

```diff
@@ -1,41 +0,0 @@
-%%%-----------------------------------------------------------------------------
-%%% @copyright (C) 2011-2018, 2600Hz
-%%% @doc
-%%% @author Karl Anderson <karl@2600hz.org>
-%%% @end
-%%%-----------------------------------------------------------------------------
--module(conference_maintenance).
```

Comment:
I would not delete the files even though they're empty. We'll need to add stuff here to interact with the conference app. In fact, there's a cache used by the app that should have a flush/0 added here! So let's keep the maintenance modules even if emptied by these changes.